src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
Good work adding tests as the previous reviewer request 🥇 

- [ ] Kindly fix this test, it's not passing

![image](https://user-images.githubusercontent.com/93630700/216190214-af0bdbd1-763b-4504-86db-483ce750a08c.png)


![image](https://user-images.githubusercontent.com/93630700/216190184-ec642b33-9330-490e-8bc1-57a0872287ea.png)